js/ui/grid_core/ui.grid_core.columns_view.js

```diff
@@ -891,6 +891,16 @@ const columnsViewMembers = {
         }
     },
 
+    _getContent: function() {
+        return this._tableElement?.parent();
+    },
+
+    _removeContent: function(isFixedTableRendering) {
+        const $scrollContainer = this._getContent(isFixedTableRendering);
+
+        $scrollContainer?.length && $scrollContainer.remove();
```

Comment:
Let's make this more obvious and add the ```if``` condition.
Also, it will help us in migration to TS because TS linter doesn't allow constructions like this :)

```suggestion
        if($scrollContainer?.length) {
          $scrollContainer.remove();
        }
```